src/@types/pkg.ts

```diff
@@ -32,7 +32,9 @@ export type TsType =
       definitelyTyped: string;
     };
 
-export type ModuleType = 'esm' | 'cjs' | 'unknown';
+export type ModuleType = 'esm' | 'cjs' | 'none' | 'unknown';
+
+export type StyleType = 'css' | 'less' | 'scss' | 'none';
```

Comment:
has this become `string` now?

Reply:
https://github.com/algolia/npm-search/pull/683/commits/80bb5089760c3fbde5e182da067ad8a9c1f49a78